vulture_whitelist.py

```diff
@@ -186,3 +186,20 @@
 LOG_DIR_NAME  # unused variable (envs/monkey_zoo/blackbox/log_handlers/test_logs_handler.py:8)
 delete_logs  # unused function (envs/monkey_zoo/blackbox/test_blackbox.py:85)
 MongoQueryJSONEncoder  # unused class (envs/monkey_zoo/blackbox/utils/json_encoder.py:6)
+
+
+# exists in common/, monkey_island/, or infection_monkey/, but is used in tests/unit_tests/ which is skipped by Vulture
+_.instance_name  # unused attribute (monkey/common/cloud/azure/azure_instance.py:35)
+_.instance_name  # unused attribute (monkey/common/cloud/azure/azure_instance.py:64)
+SomeDummyPlugin  # unused class (monkey/infection_monkey/utils/plugins/pluginTests/BadImport.py:4)
+BadPluginInit  # unused class (monkey/infection_monkey/utils/plugins/pluginTests/BadInit.py:4)
+BadInit  # unused class (monkey/infection_monkey/utils/plugins/pluginTests/ComboFile.py:4)
+ProperClass  # unused class (monkey/infection_monkey/utils/plugins/pluginTests/ComboFile.py:9)
+PluginWorking  # unused class (monkey/infection_monkey/utils/plugins/pluginTests/PluginWorking.py:4)
+environment  # unused variable (monkey/monkey_island/cc/models/monkey.py:59)
+_.environment  # unused attribute (monkey/monkey_island/cc/services/telemetry/processing/system_info_collectors/environment.py:10)
```

Comment:
Let's move these to `potentially unused`, but I think island extracts it with a mongo query somewhere and vulture is just not picking it up